src/GitHub.App/ViewModels/PullRequestListViewModel.cs

```diff
@@ -88,6 +104,24 @@ public override void Initialize([AllowNull] ViewWithData data)
                 trackingAssignees.AddItem(pr.Assignee);
                 trackingAuthors.AddItem(pr.Author);
             }, () => { });
+
+            pullRequests.OriginalCompleted
+                .ObserveOn(RxApp.MainThreadScheduler)
+                .Subscribe(_ =>
+                {
+                    if (listSettings.SelectedAuthor != null)
+                    {
+                        SelectedAuthor = Authors.FirstOrDefault(x => x.Login == listSettings.SelectedAuthor);
+                    }
+
+                    if (listSettings.SelectedAssignee != null)
+                    {
+                        SelectedAssignee = Assignees.FirstOrDefault(x => x.Login == listSettings.SelectedAssignee);
+                    }
+ 
+                    pullRequestsLoaded = true;
```

Comment:
`Initialize` is called in load and when the lost is refreshed, so this code can potentially run multiple times and you'll need to reset this flag when that happens.